sdks/python/setup.py

```diff
@@ -275,8 +274,8 @@ def get_portability_package_data():
   # executes below.
   generate_protos_first()
 
-  # Generate wrappers
-  generate_external_transform_wrappers()
+  if 'sdist' in sys.argv:
```

Comment:
what if we are building a bdist (wheel)? Why is this condition necessary?

Reply:
As discussed offline, we don't need this condition anymore. 

The root cause for this was incorrectly resolving the sdk directory path in `generate_external_transform_wrappers`, which would fail for some environments.